### DISCUSSION TITLE
UserAgent not working

### DISCUSSION THREAD

**Discussion by KebabGGbab** [cefsharp/CefSharp#4514] [General]
Hello, I would like to use CefSharp, after reading the documentation I am trying to implement the creation of multiple browser instances. But the problem is that I can't change their user agent. After I click compile project and enter any site that gives me information about my user agent, I see that it has not changed. Please tell me what could be the problem. 
Code part:

```
            var settings = new CefSettings
            {
                RootCachePath = @"D:\\program\\Projects VS\\Cookie",
                UserAgent = "Mozilla / 5.0(Windows NT 10.0; Win64; x64) AppleWebKit / 537.36(KHTML, like Gecko) Chrome / 110.0.0.0 YaBrowser / 23.3.4.603 Yowser / 2.5 Safari / 537.36"
            };
            if (Cef.IsInitialized == false)
                Cef.Initialize(settings);
            var CWBrowser = new ChromiumWebBrowser(ILink.Text)
            {
                    RequestContext = RequestContext
                    .Configure()
                    .WithProxyServer(Convert.ToString(arrayData[numberOfSelectedAccounts[i] - 1].proxyAddressAcc), Convert.ToInt32(arrayData[numberOfSelectedAccounts[i] - 1].portAcc))
                    .WithCachePath($@"D:\\program\\Projects VS\\Cookie\\{numberOfSelectedAccounts[i]}")
                    .Create(),
                    RequestHandler = new CustomRequestHandler()
           };
```
If I understand correctly, should I edit the application manifest? I added the manifest that I copied from the link: "https://github.com/cefsharp/CefSharp/blob/master/CefSharp.WinForms.Example/app.manifest", but that didn't help either. Perhaps I am doing something wrong?

**Comment by KebabGGbab** [comment]
After some actions, like the site I needed began to display the information correctly, but the UserAgent is still not displayed correctly.

**Comment by amaitland** [comment]
https://github.com/cefsharp/CefSharp/wiki/General-Usage#useragent

**Comment by jinshiyx1989** [comment]
Have you resolved it？I user the UserAgent property also not working

**Reply by KebabGGbab** [reply]
I don't remember. I stopped using Cefsharp and switched to puppeteer.
